pytests/conftest.py

```diff
@@ -52,7 +52,7 @@ def dump_cert(fn, cert):
         cert.get_subject().C = 'IN'
         cert.get_subject().ST = 'Karnataka'
         cert.get_subject().L = 'Bangalore'
-        cert.get_subject().O = 'VMware'
+        cert.get_subject().O = 'VMware'  # noqa: E741
```

Comment:
What is this?

Reply:
It's a simple way to escape a warning from flake8. `noqa` - No Questions Asked.

Reply:
I see. What's the warning though? Do they object to the "VMware" spelling?

Reply:
Some weird one, `pytests/conftest.py:55:28: E741 ambiguous variable name 'O'`